modules/nvidia-gpu-aws-deploying-the-node-feature-discovery-operator.adoc

```diff
@@ -0,0 +1,99 @@
+// Module included in the following assemblies:
+//
+//  * installing/installing-nvidia-gpu-aws.adoc
+
+:_content-type: PROCEDURE
+[id="nvidia-gpu-aws-deploying-the-node-feature-discovery-operator_{context}"]
+= Deploying the Node Feature Discovery Operator
+
+After the GPU-enabled node is created, you need to discover the GPU enabled node
+so it can be scheduled. To do this, install the Node Feature Discovery (NFD) Operator.
+
+The NFD operator identifies hardware device features in nodes.
+It solves the general problem of identifying and cataloging hardware resources in
+the infrastructure nodes so they can be made available to OpenShift.
+
+.Procedure
+
+. Install the Node Feature Discovery operator from **OperatorHub** in the
+**OpenShift Container Platform** console.
+
+. After installing the NFD operator into **OperatorHub**,
+select **Node Feature Discovery** from the installed operators list and
+select **Create instance**. This installs the ``openshift-nfd`` Operator
+into the ``openshift-operators`` namespace.
```

Comment:
The sentence is very outdated.  it should read:  This installs the "nfd-master" and "nfd-worker" pods (one nfd-worker pod for each worker node) in the "openshift-nfd" namespace.